try:
    print(n)
except:
    print("An exception occured because the variable is not assigned")
